### PR TITLE
Locations search

### PR DESCRIPTION
**Pull request by stevenwilkin**

### CODE REVIEW

**Comment by stevenwilkin** [comment]
Updated with new `Locations::SearchResult` and displaying the distance away for each rendered result

**Comment by gjvis** [comment]
:+1:

File: features/step_definitions/appointment_locator_steps.rb [hunk 1]
```diff
@@ -7,12 +8,15 @@
...
   allow(Locations::Search).to receive(:nearest_to).with(
-    locations, lat_lng, limit).and_return(locations.take(limit))
+    locations, lat_lng, limit).and_return(search_results.take(limit))
 
   Pages::Locations.new.load(postcode: postcode)
 end
 
 Then(/^I should see the (\d+) appointment locations nearest to that postcode$/) do |number_of_locations|
   expected_locations = number_of_locations.to_i.times.map { |i| "cab-#{i + 1}" }
+  expected_distances = number_of_locations.to_i.times.map { |i| "#{i}km" }
```
Comment (andrewgarner) [inline features/step_definitions/appointment_locator_steps.rb]: Why kilometres not miles?

Comment (gjvis) [inline features/step_definitions/appointment_locator_steps.rb]: Haha, I totally missed that...

Comment (stevenwilkin) [inline features/step_definitions/appointment_locator_steps.rb]: Good point heh

Comment (andrewgarner) [inline features/step_definitions/appointment_locator_steps.rb]: We should also make sure the number is rounded and we show the correct number of decimal places.

**Comment by stevenwilkin** [comment]
Updated to use miles in search results

File: features/step_definitions/appointment_locator_steps.rb (hunk 1 above)
Comment (gjvis) [inline features/step_definitions/appointment_locator_steps.rb]: How about we leave that for a presenter spec, when we implement an actual view – we don't want to be partially testing things in cucumber at this stage. Sticking with whole numbers for now keeps us moving forward with driving out the implementation.

Everyone ok with that?

Comment (andrewgarner) [inline features/step_definitions/appointment_locator_steps.rb]: :+1:

**Pull request merged**